Sources/Utilities/ALMessage+Extension.swift

```diff
@@ -345,13 +345,12 @@ extension ALMessage {
     }
 
     private func getAttachmentType() -> ALKMessageType? {
-        guard let fileMeta = fileMeta else { return nil }
         
-        var contentPrefix = String()
-        if let index = fileMeta.contentType.firstIndex(of: "/") {
-            contentPrefix = String(fileMeta.contentType.prefix(upTo: index))
-        }
+        guard let fileMeta = fileMeta, (fileMeta.contentType != nil) else { return nil }
```

Comment:
You can write a variable here and use it below like this: `let contentType = fileMeta.contentType`.

Reply:
Have added the change.

Reply:
Replace this line: `(fileMeta.contentType != nil)` with: `let contentType = fileMeta.contentType`.

Reply:
Changes done.